Sources/NIOWebSocket/WebSocketFrame.swift

```diff
@@ -363,3 +363,42 @@ extension WebSocketFrame._Storage: Equatable {
         lhs.data == rhs.data && lhs.extensionData == rhs.extensionData
     }
 }
+
+extension WebSocketFrame: CustomStringConvertible {
+    /// A `String` describing this `WebSocketFrame`. Example:
+    ///
+    ///     WebSocketFrame {
+    ///     maskKey: Optional(NIOWebSocket.WebSocketMaskingKey(_key: (187, 28, 185, 79))),
+    ///     fin: true,
+    ///     rsv1: true,
+    ///     rsv2: true,
+    ///     rsv3: true,
+    ///     opcode: WebSocketOpcode.binary,
+    ///     length: 0,
+    ///     data: ByteBuffer { readerIndex: 0, writerIndex: 0, readableBytes: 0, capacity: 0, storageCapacity: 0, slice: _ByteBufferSlice { 0..<0 }, storage: 0x00006000028246b0 (0 bytes) },
+    ///     extensionData: nil,
+    ///     unmaskedData: ByteBuffer { readerIndex: 0, writerIndex: 0, readableBytes: 0, capacity: 0, storageCapacity: 0, slice: _ByteBufferSlice { 0..<0 }, storage: 0x0000600002824800 (0 bytes) },
+    ///     unmaskedDataExtension: nil
+    ///     }
+    ///
+    /// The format of the description is not API.
+    ///
+    /// - returns: A description of this `WebSocketFrame`.
+    public var description: String {
```

Comment:
Can we please avoid adding a multiline string as a description here. `description` is often used in other interpolated strings such as log messages and adding line breaks can often break those formats. Instead can we add a single line for this.

Reply:
good point, we should add `\`s to the end of the line (such that the result is just one line). I missed that currently we'll actually print multiple lines.

Reply:
@FranzBusch very good point, thanks! I will fix it.

Reply:
This is still a multiline string from what I can see. Can you use `"` instead of `"""` unless I am missing something here

Reply:
@FranzBusch no it's not, it has `\`s everywhere but the last line.

Reply:
```
  1> print(""" 
  2.     hello \ 
  3.     my \ 
  4.     beautiful \ 
  5.     world 
  6.     """)
hello my beautiful world
```